x/stakeibc/keeper/hooks.go

```diff
@@ -41,18 +41,23 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 	}
 	// deposit records *must* exist for this epoch
 	k.SetEpochTracker(ctx, epochTracker)
+	k.Logger(ctx).Info(fmt.Sprintf("Setting epochTracker %v", epochTracker))
 
 	// process redemption records
 	if epochIdentifier == "day" {
 		// here, we process everything we need to for redemptions
 		k.Logger(ctx).Info(fmt.Sprintf("Day %d Beginning", epochNumber))
 		// first we initiate unbondings from any hostZone where it's appropriate
+		k.Logger(ctx).Info("InitiateAllHostZoneUnbondings")
```

Comment:
Extreme nit-picking, do we need this log if it's effectively covered by line 40?

Reply:
Do you mean line 49?

Reply:
Sorry, yeah

Reply:
I think leaving it will make the logs slightly more readable, so bias towards keeping it (although no strong preference)

Reply:
Let's leave it in!